Apps/Src/Voltage.c

```diff
@@ -130,12 +129,13 @@ void Voltage_OpenWireSummary(void){
  */
 SafetyStatus Voltage_OpenWire(void){
 	wakeup_idle(NUM_MINIONS);
-	long openwires = LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
-	if(openwires != 0){
-		return DANGER;
-	} else {
-		return SAFE;
+	LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
+
+	for(int32_t i = 0; i < NUM_MINIONS; i++) {
+
 	}
+
+	return DANGER;
```

Comment:
I opened an issue ticket [#76](https://github.com/lhr-solar/BPS/issues/76)  a while ago that still hasn't been worked on to fix the open wire functions. The Voltage open wire functions have never been correct. It only worked on the hardware previously because only one minion board was used during testing and not 4. I edited these functions to verify the open wire simulator and I left them like this to be updated in a different PR